src/app/features/auth/auth.component.ts

```diff
@@ -1,10 +1,77 @@
+import { NgOptimizedImage } from '@angular/common';
 import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { MatSlideToggleModule } from '@angular/material/slide-toggle';
+
+import { FormComponent } from './form/form.component';
 
 @Component({
   selector: 'app-auth',
   standalone: true,
-  template: ` <p>auth works!</p> `,
-  styles: [],
+  imports: [
+    MatSlideToggleModule,
+    NgOptimizedImage,
+    FormComponent
+  ],
+  template: `
+    <main>
+      <div class="dark-mode-toggle">
+        <mat-slide-toggle></mat-slide-toggle>
+      </div>
+      <div class="form-wrapper">
+        <div>
+          <img ngSrc="assets/images/logo.svg" alt="logo" width="300" height="150" priority />
+        </div>
+        <app-form></app-form>
+      </div>
+    </main>
+  `,
+  styles: [
+    `
+      main {
+        background-color: var(--background);
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 100vh;
+      }
+
+      .dark-mode-toggle {
+        position: absolute;
+        top: 20px;
+        right: 20px;
+      }
+
+      .form-wrapper {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        background-color: var(--surface);
+        width: 450px;
+        height: 550px;
+        box-shadow: 0 0 4px 0 rgba(0, 0, 0, 0.1);
+      }
+
+      .input {
+        width: 300px;
+        height: 50px;
+      }
+
+      @media screen and (max-width: 568px) {
+        .form-wrapper {
+          width: 100vw;
+          height: 100vh;
+        }
+      }
+
+      @media screen and (max-width: 1024px) {
```

Comment:
nie mamy tych breakpointów w zmiennych w design systemie? jeśli nie to umiescmy je tam i wykorzystajmy scss by nie hardcodować tego tylko uzyc wartosciw  zmiennej

https://bholmes.dev/blog/alternative-to-css-variable-media-queries/